Sources/Host/POSIX/ProcessSpawner.cpp

```diff
@@ -611,9 +611,8 @@ void ProcessSpawner::redirectionThread() {
         hup = true;
       }
 
-      size_t index;
       RedirectDescriptor *descriptor = nullptr;
-      for (index = 0; index < 3; index++) {
+      for (int index = 0; index < 3; index++) {
```

Comment:
`size_t` is not required and `int` is shorter to type.

On Tue, Jul 5, 2016 at 12:21 PM, kokarez <notifications@github.com> wrote:

> In Sources/Host/POSIX/ProcessSpawner.cpp
> <https://github.com/facebook/ds2/pull/397#discussion_r69622185>:
>
> >        RedirectDescriptor *descriptor = nullptr;
> > -      for (index = 0; index < 3; index++) {
> > +      for (int index = 0; index < 3; index++) {
>
> Any reason size_t -> int ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/facebook/ds2/pull/397/files/20dc03737a25ae7c3beb361b86444f83bc50feb6#r69622185>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AADHbD2O2viinDxPEi17Lt1otHMXGyqHks5qSq7CgaJpZM4JFbLD>
> .
>



-- 
Stephane Sezer